datajunction-server/datajunction_server/api/nodes.py

```diff
@@ -278,13 +273,13 @@ def list_all_nodes_with_details(
     return [row for row in results if row.name in approvals]
 
 
-@router.get("/nodes/{name}/", response_model=NodeOutput)
+@router.get("/nodes/{name}/")
```

Comment:
The response still looks like a `NodeOutput` instance so shouldn't `response_model=NodeOutput` be kept here?

Reply:
Adding `response_model=NodeOutput` actually causes FastAPI to use the Pydantic model to parse the output twice, which fails because we have some custom logic on `NodeOutput` that "flattens" a node revision into the output model. This flattening logic should be removed, but I'm a bit wary of doing that here because this PR is already quite big.

Reply:
Adding `response_model=NodeOutput` actually causes FastAPI to use the Pydantic model to parse the output twice, which fails because we have some custom logic on `NodeOutput` that "flattens" a node revision into the output model. This flattening logic should be removed, but I'm a bit wary of doing that here because this PR is already quite big.

Reply:
Ok that's fair--agreed that's better left to a follow up PR